app/src/main/res/layout/fragment_explore_map.xml

```diff
@@ -62,6 +62,7 @@
     android:padding="@dimen/activity_margin_horizontal"
     android:singleLine="true"
     android:text="@string/search_this_area"
+    android:textAllCaps="false"
```

Comment:
What is the problem if you don't add this line?

Reply:
The line was initially included to ensure that the button text aligned with the original TextView string (which has now been removed) as Android Studio defaults buttons to AllCaps. However, since we are removing it, it may no longer be necessary.

Reply:
How about removing this whole button?

Reply:
While the TextView that used to prompt users to use this button has been refactored and removed, the button remains in the app and is still used in the Map Explore activity, as shown below ![button_usage](https://github.com/user-attachments/assets/14a1f000-9263-4e1d-bdb5-e1cab8976358)

Reply:
Do you know what makes the text become uppercase in the first place?
We should fix the root issue rather than address this single symptom.
Would you mind checking whether these are really needed?
```
app/src//main/res/layout-land/activity_review.xml:        android:textAllCaps="true"
app/src//main/res/layout/bottom_sheet_details_explore.xml:            android:textAllCaps="true"
app/src//main/res/layout/bottom_sheet_details_explore.xml:            android:textAllCaps="true"
app/src//main/res/layout/bottom_sheet_details_explore.xml:          android:textAllCaps="true"
app/src//main/res/layout/fragment_achievements.xml:            android:textAllCaps="true"/>
app/src//main/res/layout/fragment_achievements.xml:            android:textAllCaps="true"/>
app/src//main/res/layout/custom_nearby_tab_layout.xml:        android:textAllCaps="true"
app/src//main/res/layout/activity_review.xml:      android:textAllCaps="true"
app/src//main/res/layout/fragment_review_image.xml:      android:textAllCaps="true"
app/src//main/res/layout/fragment_review_image.xml:      android:textAllCaps="true"
```